README.md

```diff
@@ -8,13 +8,73 @@
 + Delete a key-value pair from an object
 + Iterate over key-value pairs in an object
 
-
 ## What the Student Already Knows
 
 - Variables
 - Data Type- Strings and Integers
 - Functions
-- Booleans, Comparison Operators, Flow Control 
+- Booleans, Comparison Operators, Flow Control
 - Arrays
 
+##Understanding Objects
+
+JavaScript objects are a set of key-value pairs. Objects use the JavaScript object notation, for example, `{ 'key': 'value' }`. Each key inside an object may have any value type. For example, strings, integers, booleans, and functions. JavaScript does not care what the value is as long as it maps to a key in the set.
+
+To create an object, fire up a plain old REPL in Node.js. A REPL is a Read Eval Print Loop, and I invite you to type right along with me. Letʼs write this like so:
+
+```javascript
+> var o = { key1: 1 };
+```
+
+Notice in this example I omit the single quotes around the key when creating this object. JavaScript is not picky about syntax. For objects with special characters in the key, put single or double quotes around it. For example, `{ 'my-special-key': 'value' }`. I prefer to use single quotes, the choice is yours.
+
+###Accessing Objects
+
+To access the object I created above in the REPL, do this:
+
+```javascript
+> o.key1;
+```
+
+In JavaScript, one can use the dot or bracket notation to do a lookup on the object. To use the bracket notation, for example, do a `o['key1']`. Again, JavaScript is not picky on how you do it. Each technique solves a different problem, like in the case of special characters on the key. To access keys with special characters, make sure there are quotes around it. For example, `o['my-special-key']`.
+
+###Adding a Key-Value Pair
+
+To add a key-value pair in the REPL, type up:
+
+```javascript
+> o.key2 = 2;
+```
+
```

Comment:
in general, using key1 and 1 is a bit abstract. can we use actual examples that serve a narrative?